src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -85,7 +86,7 @@ public function getPriority()
      */
     public function isCandidate(Tokens $tokens)
     {
-        return $tokens->isTokenKindFound(T_FUNCTION) && (
+        return $tokens->findGivenKind([T_FUNCTION, T_CLASS]) && (
```

Comment:
I think this fixer should work for interfaces as well that extend other interfaces?

Reply:
Yes, updated now.